Tribler/community/multichain/.gitignore

```diff
@@ -0,0 +1 @@
+.Tribler-*
```

Comment:
If your test is writing stuff here it's broken. Use the statedir path on the base test class (which is a temporary one that gets cleaned up after the test finishes)